### PR TITLE
Docs review updated 2024-03-monthly.md

### PR DESCRIPTION
**Pull request by oduameh**

### CODE REVIEW

File: monthly/2024-03-monthly.md [hunk 1]
```diff
@@ -73,78 +73,78 @@ change much this month:
...
+SundaeLabs is working on a Hydra-based layer two protocol (`gummiworm`) that requires the usage of `hydra-node` components in a different way than the default setup. As part of this development, SundaeLabs submitted a Fund10 Catalyst proposal to enhance the Hydra node by adding features that would facilitate not only their work but also the work of everyone building on Hydra.
 
-They have been working on a plugin architecture for exfiltrating and infiltrating events, which enables hygienic forks of Hydra that add different connections to the world. In the case of SundaeLabs, they are interested in consuming the Hydra event stream through Amazon Kinesis and store transaction archives on S3 for later reference.
+The team has been working on a plugin architecture to enable the infiltration and exfiltration of events, which enables hygienic forks of Hydra that feature additional connections to the outside world. SundaeLabs, for instance, is interested in consuming the Hydra event stream via Amazon Kinesis, and storing transaction archives on S3 for later reference.
```
Comment (ch1bo) [inline monthly/2024-03-monthly.md:113]: @oduameh Here I was referencing the SundaeLabs team with `they`. Should we be more inclusive and call anyone working on Hydra or Mithril as `the team`?

File: monthly/2024-03-monthly.md [hunk 2]
```diff
@@ -73,78 +73,78 @@ change much this month:
...
 
 ## Community
 
 ### Mithril office hours
 
-As the catalyst project of TxPipe on Mithril is completed, we want to build upon this initial maintenance work in a more open format. For this, we are transforming our alignment calls into a public weekly office hour where we invite the community to join and discuss concrete pull requests, issues and questions on Mithril. 
+The Catalyst project of TxPipe on Mithril has been completed by the team, and they want to continue building on this initial maintenance work in a more open format. To achieve this, the team is transforming their alignment calls into a public weekly office hour where they invite the community to join and discuss concrete pull requests, issues, and questions related to Mithril.
```
Comment (ch1bo) [inline monthly/2024-03-monthly.md:121]: @oduameh I specifically chose a passive tone here and `TxPipe` was the subject of completing the project (it's theirs). It is a related point as I commented above. In both cases, I wanted to specifically say that SundaeLabs/TxPipe has been contributing (to underline the fact that it's not only IOG people working on this).

How should we make this align with this all `The team`-speak?

**Review by ch1bo — COMMENTED** [review]
I'm not sure about this to be honest. 

80% of theses proposed changes are changing "we" into "they". Personally, I don't like this far removed tone of "The team" and "they" all the time. I would much prefer the more inclusive "We" in most cases.

But thanks for the other 20% of suggestions!

So what do you suggest in how we handle this?

File: monthly/2024-03-monthly.md (hunk 1 above)
Comment (olgahryniuk) [inline monthly/2024-03-monthly.md:113]:
```suggestion
SundaeLabs have been working on a plugin architecture to enable the infiltration and exfiltration of events, which enables hygienic forks of Hydra that feature additional connections to the outside world. They are interested in consuming the Hydra event stream via Amazon Kinesis and storing transaction archives on S3 for later reference.
```

File: monthly/2024-03-monthly.md (hunk 2 above)
Comment (olgahryniuk) [inline monthly/2024-03-monthly.md:121]:
```suggestion
As the Catalyst project of TxPipe on Mithril has been completed, Hydra and Mithril projects aim to continue building on this initial maintenance work in a more open format. To achieve this, the scaling teams are transforming their alignment calls into a public weekly office hour where they invite the community to join and discuss concrete pull requests, issues, and questions related to Mithril. 
```

**Review by olgahryniuk — APPROVED** [review]

**Review by ch1bo — APPROVED** [review]
I like it!

**Comment by ch1bo** [comment]
Squashed and rebased

**Pull request merged**